api/db/populate-messages.js

```diff
@@ -0,0 +1,55 @@
+const { getArticle } = require('./utils');
+
+const Friendship = require('../models/Friendship');
+const Message = require('../models/Message');
+
+let displayLogs;
+
+const createMessage = async (messageParams) => {
+  const message = new Message({
+    friendship: messageParams.friendship,
+    user: messageParams.user,
+    content: messageParams.content,
+  });
+  message.save();
+
+  if (displayLogs) {
+    console.log('\n', '\x1b[0m', `New message created: ${message}`);
+  }
+  return message;
+};
+
+const createMessages = async (verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating messages collection...');
+  displayLogs = verbose;
+
+  const friendshipsArray = await Friendship.find();
+  const messagesArray = [];
+
+  for (const friendship of friendshipsArray) {
+    const messagesCount = Math.floor(Math.random() * 10);
```

Comment:
I would say we should make this number bigger and instead make the content of every message smaller, as that could be a better model a real conversation? I'll leave a comment about this where the content of the message is generated too

Reply:
That's fine. However, I would suggest we reconsider the quantity of users and/or events we are creating. It is beginning to take a very long time to generate all the collections. And most of these items we only really need a handful of records to get a clear idea of how the application will behave.

Reply:
Yeah, I think we can reduce those by a factor of ten, and if someone needs more they can just use the `-m` argument for that